### ISSUE TITLE
Removing writemultiple operation generation for http

### ISSUE THREAD

**Issue by egekorkan** [eclipse-thingweb/node-wot#1087]
> However, wouldn't it be better to never generate a TD with it?

Good point, I removed it for now alongside the `handleWriteMultipleProperties` method in order to not confuse people. However, note that in the HTTP server, the write meta operations are actually added to the `op` array despite having no corresponding handler method defined:

https://github.com/eclipse-thingweb/node-wot/blob/227be532bc98f5f6c8aeb4abc9de07cccca0ce71/packages/binding-http/src/http-server.ts#L346-L366

_Originally posted by @JKRhb in https://github.com/eclipse-thingweb/node-wot/pull/1086#issuecomment-1721530665_

**Comment by danielpeintner** [comment]
@relu91 what's your opinion? Should we simply remove the "multiple" op's ?

**Comment by relu91** [comment]
Yes I think we should remove it as long as we don't have a clear picture on how to handle this in the TD spec.
